Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -74,7 +74,97 @@ TEST_CASE("tag atoms in SVG", "[drawing][SVG]") {
     CHECK(text.find("atom-selector") != std::string::npos);
     CHECK(text.find("bond-selector") != std::string::npos);
   }
+  SECTION("inject prop to class") {
+    auto m1 = "C1N[C@@H]2OCC12"_smiles;
+    REQUIRE(m1);
+
+    for (auto it = m1->beginAtoms(); it != m1->endAtoms(); it++) {
+      auto prop = boost::format("__prop_class_atom_%d") % (*it)->getIdx();
+      (*it)->setProp("_tagClass", prop.str());
+    }
```

Comment:
A bit simpler and easier to read
```suggestion
    for (auto atom : m1->atoms() ) {
      auto prop = boost::format("__prop_class_atom_%d") % atom->getIdx();
      atom->setProp("_tagClass", prop.str());
    }
```

Reply:
Changed